protobuf/src/core.rs

```diff
@@ -108,6 +108,8 @@ pub trait Message: fmt::Debug + Clear + Send + Sync + ProtobufValue {
     }
 
     /// Write the message to bytes vec.
+    
+    /// > **Note**: You can use parse_from_bytes to do the reverse.
```

Comment:
Thank you for the PR!

Can you please add `///` on the previous empty line and backtricks around `parse_from_bytes`?